forge-game/src/main/java/forge/game/trigger/TriggerDamageDoneOnce.java

```diff
@@ -7,6 +7,7 @@
 
 import forge.game.ability.AbilityKey;
 import forge.game.card.Card;
+import forge.game.card.CardCollection;
```

Comment:
you also need to remove line 4 and 6 for this to pass the tests

Reply:
Thanks, I'd missed that. Tests are now passing